docs/4.2/enterprise/quickstart-enterprise.md

```diff
@@ -47,7 +47,7 @@ To start using Teleport Enterprise, you will need to Download the binaries and t
 After downloading the binary tarball, run:
 
 ```bsh
-$ tar -xzf teleport-ent-v{{ teleport.version }}-linux-amd64-bin.tar.gz
+$ tar -xzf <tar.gz>
```

Comment:
Was there a specific reason to change this? I recently updated this so the version number will propagate to the UI and it'll look like this when rendered. 

![image](https://user-images.githubusercontent.com/559288/75589396-7e785700-5a2f-11ea-9c1d-5fb0521da664.png)

Reply:
Yes, please bump it to 4.2.3. It's set here, https://github.com/gravitational/teleport/blob/master/docs/4.2.yaml#L35 
Please also update the SHA for OSS to `1c14362c9ba10f28088c7228b357dc6a70072d3d4afaa5510c70a8734068684c`

Reply:
Awesome, will do!